internal/cmd/configuration/command.go

```diff
@@ -0,0 +1,74 @@
+package configuration
+
+import (
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/config"
+)
+
+type command struct {
+	*pcmd.CLICommand
+	config *config.Config
+}
+
+type configurationOut struct {
+	Name     string `human:"Name" serialized:"name"`
+	Value    string `human:"Value" serialized:"value"`
+	ReadOnly bool   `human:"Read-Only" serialized:"read_only"`
+}
+
+func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "configuration",
+		Aliases: []string{"config"},
+		Short:   "Manage CLI configuration fields.",
+	}
+
+	c := &command{
+		CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner),
+		config:     cfg,
+	}
+
+	cmd.AddCommand(c.newSetCommand())
+	cmd.AddCommand(c.newListCommand())
+
+	return cmd
+}
+
+func getSettableConfigFields(cfg *config.Config) (map[string]reflect.Kind, map[string]string) {
+	jsonFieldToType := make(map[string]reflect.Kind)
+	jsonFieldToName := make(map[string]string)
+	t := reflect.TypeOf(*cfg)
+	for i := 0; i < t.NumField(); i++ {
+		field := t.Field(i)
+		kind := field.Type.Kind()
+		switch kind {
+		case reflect.Bool, reflect.String:
+			jsonFieldName := getJsonFieldName(field)
+			if jsonFieldName != "" {
+				jsonFieldToType[jsonFieldName] = kind
+				jsonFieldToName[jsonFieldName] = field.Name
+			}
+		}
+	}
+	return jsonFieldToType, jsonFieldToName
+}
+
+func getJsonFieldName(field reflect.StructField) string {
+	jsonTag := field.Tag.Get("json")
+	if jsonTag == "-" {
+		return ""
+	}
+	if strings.Contains(jsonTag, ",") {
+		jsonTag, _, _ = strings.Cut(jsonTag, ",")
+	}
+	if runtime.GOOS != "windows" && jsonTag == "disable_plugins_once" {
```

Comment:
Any way we can get around hardcoding this specific case?

Reply:
I don't think so unfortunately. Hardcoding this case makes it so that `disable_plugins_once` doesn't show up in the output of `config list` and is not settable unless the user is running Windows. 
If a Mac/Unix user saw this in `config list`, they could reasonably infer that setting this flag would disable plugins one time, but in actuality setting it would do nothing.
Maybe we should instead append a `_windows` to this config field to avoid confusion?

Reply:
Ah, makes sense.

> Maybe we should instead append a _windows to this config field to avoid confusion?

That's a good idea, but doing so would be a breaking change. Feel free to open a ticket for this under "v4" though!